mail/postfix/src/opnsense/mvc/app/controllers/OPNsense/Postfix/forms/general.xml

```diff
@@ -81,7 +101,7 @@
     </field>
     <field>
         <id>general.permit_mynetworks</id>
-        <label>Permit My Networks</label>
+        <label>Permit Trusted Networks</label>
```

Comment:
Why do we need permit checkbox and multiselect trusted networks? Set default, required to N?

Reply:
@fichtner the multiselect is for postfix. This settings are used at multiple places while this checkbox is for the restrictions (ACL).